Makefile

```diff
@@ -275,7 +275,8 @@ apply-crds:
 	kubectl apply \
 	-f $(APPLICATIONS_CRD) \
 	-f $(COMPONENT_DETECTION_QUERIES_CRD) \
-	-f $(COMPONENT_CRD)
+	-f $(COMPONENT_CRD) \
+	-f https://raw.githubusercontent.com/redhat-appstudio/managed-gitops/main/appstudio-shared/manifests/appstudio-shared-customresourcedefinitions.yaml
```

Comment:
Should the `install` makefile target be updated to apply the shared CRDs? Then we can cut out the section in the readme about installing the crd.

Reply:
updated